ui/raidboss/data/triggers/o5s.js

```diff
@@ -14,48 +54,158 @@
     {
       id: 'O5S Doom Strike',
       regex: /14:28B1:Phantom Train starts using Doom Strike on (\y{Name})/,
+	  regexDe: /14:28B1:Phantomzug starts using Vernichtungsschlag on (\y{Name})/,
+	  //regexFr: /14:28B1:/,
+	  //regexKo: /14:28B1:/,
+	  
       alertText: function(data, matches) {
         if (matches[1] == data.me)
-          return 'Tank Buster on YOU';
+          return {
+			  en: 'Tank Buster on YOU', 
+			  de: 'Tank Buster auf DIR',
+			  fr: 'Tank Buster on YOU',
+			  ko: 'Tank Buster on YOU',
+			  };
         if (data.role == 'healer')
-          return 'Buster on ' + data.ShortName(matches[1]);
+          return {
+			  en: 'Buster on ' + data.ShortName(matches[1]),
+			  de: 'Buster auf ' +data.ShortName(matches[1]),
+			  fr: 'Buster on ' + data.ShortName(matches[1]),
+			  ko: 'Buster on ' + data.ShortName(matches[1]),
+			  };
       },
       tts: function(data, matches) {
         if (matches[1] == data.me)
-          return 'buster';
+          return {
+			  en: 'buster',
+			  de: 'Buster',
+			  fr: 'buster',
+			  ko: 'buster',
+			  };
       },
     },
     {
       id: 'O5S Head On',
       regex: /14:28A4:Phantom Train starts using Head On/,
-      alertText: 'Go To Back',
-      tts: 'run away',
+	  regexDe: /14:28A4:Phantomzug starts using Frontalangriff/,
+	  //regexFr: /14:28A4:/,
+	  //regexKo: /14:28A4:/,
+	  
+      alertText: function (data) {
+		  return {
+			  en: 'Go To Back',
+			  de: 'Nach HINTEN!',
+			  fr: 'Go To Back',
+			  ko: 'Go To Back',
+			  };
+	  },
+      tts: function(data) {
+		  return {
+			  en: 'run away',
+			  de: 'renn weg',
+			  fr: 'run away',
+			  ko: 'run away',
+		  };
+	  },
     },
     {
       id: 'O5S Diabolic Headlamp',
       regex: /14:28B2:Phantom Train starts using Diabolic Headlamp/,
-      alertText: 'Stack Middle',
-      tts: 'stack middle',
+	  regexDe: /14:28B2:Phantomzug starts using Diabolische Leuchte/,
+	  //regexFr: /14:28B2:/,
+	  //regexKo: /14:28B2:/,
+	  
+	  alertText: function (data) {
+		  return {
+			  en: 'Stack Middle',
+			  de: 'Versammeln Mitte',
+			  fr: 'Stack Middle',
+			  ko: 'Stack Middle',
+			  };
+	  },
+      tts: function(data) {
+		  return {
+			  en: 'stack middle',
+			  de: 'versammeln mitte',
+			  fr: 'stack middle',
+			  ko: 'stack middle',
+		  };
+	  },
+
     },
     {
       id: 'O5S Diabolic Light',
       regex: /1B:........:(\y{Name}):....:....:0001:0000:0000:0000:/,
+	  regexDe: /1B:........:(\y{Name}):....:....:0001:0000:0000:0000:/,
```

Comment:
If the regexes are the same in other languages, you can just specify regex and not need to specify regexDe/regexFr/regexKo.  I think this would reduce duplication here.

Reply:
realized this in o6s and following